--- src/training/steps/model_training/analyst_base_config.yaml	2025-12-09 11:25:03.066677653 +0000
+++ src/training/steps/model_training/analyst_base_config.yaml_saved	2025-12-09 11:25:03.070678115 +0000
@@ -27,8 +27,8 @@
         reg_lambda: 1.1236203565420875
         sampling_rate: 0.9852142919229748
         min_child_samples: 64
-        objective: regression
-        metric: rmse
+        objective: regression_l1
+        metric: mae
         early_stopping_rounds: 100
         min_boost_rounds: 200
       hpo:
@@ -118,8 +118,8 @@
         reg_lambda: 1.1236203565420875
         sampling_rate: 0.9852142919229748
         min_child_samples: 64
-        objective: regression
-        metric: rmse
+        objective: regression_l1
+        metric: mae
         early_stopping_rounds: 100
         min_boost_rounds: 200
     knn:
@@ -202,7 +202,7 @@
       use_isotonic_calibration: true
       output_mean_and_variance: true
       params:
-        n_estimators: 500
+        n_estimators: 300
         learning_rate: 0.01
         minibatch_frac: 0.2
         verbose: false
